### PR TITLE
update FAQ for tensorflow and py3.9

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)
Apply (in spirit) the proposed changes from https://github.com/Unity-Technologies/ml-agents/pull/4822 and https://github.com/Unity-Technologies/ml-agents/pull/4823.

Thanks to @stickseloni for noticing these.

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)
https://github.com/Unity-Technologies/ml-agents/pull/4822 and https://github.com/Unity-Technologies/ml-agents/pull/4823.

### Types of change(s)
- [x] Documentation update

### Checklist
- [x] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)

### Other comments

Note that this PR targets the release branch; since it's just a documentation change, I'll update the `release_12_docs` tag after merging. The relevant section of the FAQ was removed from master on https://github.com/Unity-Technologies/ml-agents/pull/4826.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ will try to install it when you install the ml-agents package.
 If you see a message like this
 
 ```console
-ERROR: Could not find a version that satisfies the requirement tensorflow<2.0,>=1.7 (from mlagents) (from versions: none)
-ERROR: No matching distribution found for tensorflow<2.0,>=1.7 (from mlagents)
+ERROR: Could not find a version that satisfies the requirement tensorflow<3.0,>=1.14 (from mlagents) (from versions: none)
```
Comment (chriselion) [inline docs/FAQ.md:13]: This is the range for `mlagents[tensorflow]`

File: docs/FAQ.md [hunk 2]
```diff
@@ -20,17 +20,21 @@ Some known potential causes are:
 - You're using 32-bit python instead of 64-bit. See the answer
   [here](https://stackoverflow.com/a/1405971/224264) for how to tell which you
   have installed.
+- You're using python 3.9. TensorFlow plans to release packages for this as soon
```
Comment (chriselion) [inline docs/FAQ.md:23]: Revive the 3.8 wording from https://github.com/Unity-Technologies/ml-agents/commit/6eca1f2e828b45bf25c20686191a726c1a6456fc#diff-0bb42a85d04c8b7c9b659307979fd43e8f6e084283a300bfff959adfcbddfd17

**Comment by stickseloni** [comment]
@chriselion that is fine with me, thank you

**Review by ervteng — APPROVED** [review]

**Comment by chriselion** [comment]
[`release_12_docs`](https://github.com/Unity-Technologies/ml-agents/tree/release_12_docs) tag updated

**Pull request merged**